apps/ledger-live-mobile/src/icons/TachometerFast.tsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import Svg, { Path } from "react-native-svg";
+
+type Props = {
```

Comment:
2 questions:

- Why making these props mandatory as they have default values in the component
- Couldn't we make this a shared `IconProps` used by all icons ?

Reply:
- Added with the memoization 👌 
- That would be a live-hub topic, I don't know how feasible or not this is, especially now that we're building new icons library I think.